source/core/timeseries/timeseries-limitations.txt

```diff
@@ -117,11 +126,14 @@ Client-Side Field Level Encryption
 </core/security-client-side-encryption>` is not supported for
 :ref:`time series collections <manual-timeseries-collection>`.
 
+.. _time-series-limitations-sharding:
+
 Sharding
 ~~~~~~~~
 
-:ref:`Time series collections <manual-timeseries-collection>` cannot
-currently be sharded.
+Starting in MongoDB 5.1, sharded time series collections are supported.
```

Comment:
We should probably add more details here. Some of the key things to highlight would be,

1. The changes the the `shardCollection` command, which now accepts `timeseries` parameter.
2. The sharding admin commands (like split, moveChunk) are not supported for time-series collection. 
3. The limitations on the shard key pattern (Can only use `metaField` and `timeField` in the shard key).